clients/tests/unit/render/Mocks/mockUtils.js

```diff
@@ -0,0 +1,208 @@
+	size = size || 1024;
+	mimeType = mimeType || 'plain/txt';
+
+	function range(count) {
+		var output = "";
+		for (var i = 0; i < count; i++) {
+			output += "a";
+		}
+		return output;
+	}
+
+	var blob = new Blob([range(size)], {type: mimeType});
+	blob.lastModifiedDate = new Date();
+	blob.name = name;
+
+	return blob;
+};
+
+const size = 1024 * 1024 * 2;
+const mock = new MockFile();
+export const file = mock.create("pic.png", size, "image/png");
+
+export const threeFiles = [file, file, file];
+
+export const CF_API_DATA = {
+	"rest": {
+		"root": "http://localhost:8228/wp-json/cf-api/v2/",
+		"rootV3":"http://localhost:8228/wp-json/cf-api/v3/",
+		"fileUpload":"http://localhost:8228/wp-json/cf-api/v3/file",
+		"tokens": {
+			"nonce":"http://localhost:8228/wp-json/cf-api/v2/tokens/form"
+		},
+		"nonce":
+			"f89f76cda3"
+	},
+	"strings": {
+		"cf2FileField": {
+			"removeFile":"Remove file",
+			"defaultButtonText":"Drop files or click to select files to Upload"
+		}
+	},
+	"nonce":{
+		"field":"_cf_verify"
+	}
+};
+export const CFFIELD_CONFIG = {"1":{"configs":{"fld_4065869":{"type":"summary","id":"fld_4065869_1","default":"","form_id":"CF5bc8e4db50622","form_id_attr":"caldera_form_1","sync":true,"tmplId":"html-content-fld_4065869_1-tmpl","contentId":"html-content-fld_4065869_1","bindFields":[{"tag":"{{fld_5843941}}","to":"fld_5843941_1"},{"tag":"{{fld_6540633}}","to":"fld_6540633_1"},{"tag":"{{fld_9226671}}","to":"fld_9226671_1"},{"tag":"{{fld_990986}}","to":"fld_990986_1"},{"tag":"{{fld_7276122}}","to":"fld_7276122_1"}]},"fld_6660137":{"type":"button","id":"fld_6660137_1","default":"","form_id":"CF5bc8e4db50622","form_id_attr":"caldera_form_1"}},"fields":{"ids":["fld_5843941_1","fld_6540633_1","fld_4065869_1","fld_9226671_1","fld_990986_1","fld_6660137_1","fld_7276122_1"],"inputs":[{"type":"text","fieldId":"fld_5843941","id":"fld_5843941_1","options":[],"default":""},{"type":"text","fieldId":"fld_6540633","id":"fld_6540633_1","options":[],"default":"old default"},{"type":"summary","fieldId":"fld_4065869","id":"fld_4065869_1","options":[],"default":""},{"type":"cf2_file","fieldId":"fld_9226671","id":"fld_9226671_1","options":[],"default":""},{"type":"dropdown","fieldId":"fld_990986","id":"fld_990986_1","options":[],"default":"No"},{"type":"button","fieldId":"fld_6660137","id":"fld_6660137_1","options":[],"default":""},{"type":"text","fieldId":"fld_7276122","id":"fld_7276122_1","options":[],"default":""}],"groups":[],"defaults":{"fld_5843941_1":"","fld_6540633_1":"old default","fld_4065869_1":"","fld_9226671_1":"","fld_990986_1":"No","fld_6660137_1":"","fld_7276122_1":""},"calcDefaults":{"fld_5843941_1":0,"fld_6540633_1":0,"fld_4065869_1":0,"fld_9226671_1":0,"fld_990986_1":0,"fld_6660137_1":0,"fld_7276122_1":0}},"error_strings":{"mixed_protocol":"Submission URL and current URL protocols do not match. Form may not function properly.","jquery_old":"An out of date version of jQuery is loaded on the page. Form may not function properly."}}};
```

Comment:
@New0 I'm happy to see you're devloping reusable test data. In Caldera 2.0, I'm starting a convention of having files for these called <description>.fixtures.js for example fields.fixtures.js supplies field configurations for testing https://github.com/Shelob9/caldera/blob/master/js-packages/forms/src/components/CalderaForm/fields.fixtures.js